### DISCUSSION TITLE
教科書AutoLayoutにて、Previewが見つかりません、、

### DISCUSSION THREAD

**Discussion by sheep1857** [lit-iphone-course/lit-iphone-course#132] [Question-original] [answered]
## (1) 実現したいこと
Previewを表示したい
## (2) 実現するために自分が試した内容とその結果
教科書に沿って右上部のバーから探したり、[option]+[command]+[enter]を押したが、表示されない。
## (4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)
<img width="1453" alt="スクリーンショット 2023-02-09 2 08 54" src="https://user-images.githubusercontent.com/105029351/217601958-d4f629aa-7d5d-419b-a110-41de348fab85.png">

## (5) GitHubのリンク

**Comment by txxxxc** [comment]
Xcode14からPreviewが削除されてしまったのが理由です！
まだ教科書のポータルサイトの方に反映ができていないのですが、アップデートされた資料は完成しているのでDMで送ります！🙌

**Comment by nokonoko2057** [comment, marked as answer]
@sheep1857 @txxxxc 修正した教科書をポータルサイトにアップロードしました！
そちらで確認お願いします！
